### PR TITLE
chore: Update hashgraph-transfer.md

### PR DESCRIPTION
**Pull request by rbarkerSL**
Update the hashgraph-transfer.md file with
- Pre-transfer checks
- Day-before checks
- Updated day-of checks
- Updated Post-transfer checks

### CODE REVIEW

**Review by andrewb1269hg — APPROVED** [review]

File: hashgraph-transfer.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Hashgraph to Hiero Repository Transfer Process
 
-When transferring ownership of a repository from [Hashgraph](https://github.com/hashgraph) to [Hiero-Ledger](https://github.com/hiero-ledger) in Github the steps outlined below must be completed
+When transferring ownership of a repository from [Hashgraph][hashgrah] to [Hiero-Ledger][hiero] in Github the steps outlined below must be completed
```
Comment (andrewb1269hg) [inline hashgraph-transfer.md:3]: Misspelling of `hashgraph` here.

File: hashgraph-transfer.md [hunk 2]
```diff
@@ -13,33 +13,80 @@ When transferring ownership of a repository from [Hashgraph](https://github.com/
...
+## Day Before Checks
+
+The day before a repository is transferred the TA and AA shall verify the following checks have been complete
+
+- DCO Audit is complete
+- License Audit is complete
+- Pre-Transfer Checks are complete
+  - TA has the following:
+    - New repository Information
+    - List of MC packages
```
Comment (andrewb1269hg) [inline hashgraph-transfer.md:57]: I would spell out `Maven Central` here instead of `MC`

File: hashgraph-transfer.md [hunk 3]
```diff
@@ -13,33 +13,80 @@ When transferring ownership of a repository from [Hashgraph](https://github.com/
...
+    - List of Gradle Plugin Central packages
+    - List of NPMJS packages
+    - Awareness of Codecov needs
+    - List of repository variables/states
+    - List of repository secrets
+    - Repository Point of Contact for Day of questions
```
Comment (andrewb1269hg) [inline hashgraph-transfer.md:63]: Change from `Day of` to `day-of`

File: hashgraph-transfer.md [hunk 4]
```diff
@@ -13,33 +13,80 @@ When transferring ownership of a repository from [Hashgraph](https://github.com/
...
+- Developer Experience team is allerted that repository is ready to transfer
```
Comment (andrewb1269hg) [inline hashgraph-transfer.md]: `alerted` typo.

File: hashgraph-transfer.md [hunk 5]
```diff
@@ -13,33 +13,80 @@ When transferring ownership of a repository from [Hashgraph](https://github.com/
...
+
+The Developer Experience Team shall announce the upcoming repository transfer to the community once the TA and AA verify
+the repository is ready for transfer
```
Comment (andrewb1269hg) [inline hashgraph-transfer.md:67]: Add a period at the end of this sentence.

File: hashgraph-transfer.md [hunk 6]
```diff
@@ -9,37 +9,84 @@ When transferring ownership of a repository from [Hashgraph](https://github.com/
...
+  - List of currrently published packages (from [Hashgraph][hashgraph]) is provided to **Repository Transfer Agent**
+- If codecov artifacts are captured for the repository
+  - **Repository Transfer Agent** is made aware
+
+- The [@hashgraph/devops-ci team][devops-ci] will capture the repository variables and variable states and provide to
+  the TA prior to day of transfer
+- The [@hashgraph/devops-ci team][devops-ci] will capture the repository secrets and provide this list to the TA prior
+  to day of transfer
+
+## Day Before Checks
```
Comment (andrewb1269hg) [inline hashgraph-transfer.md:48]: `Day-Before` (add a hyphen)

File: hashgraph-transfer.md [hunk 7]
```diff
@@ -9,37 +9,84 @@ When transferring ownership of a repository from [Hashgraph](https://github.com/
...
-    - The `REPO-maintainers` group should have the `maintain` role on the repository
-    - The `REPO-committers` group should have the `write` role on the repository
-- The AA will approve the changes in `Hiero-Ledger/governance`
+- The TA will open a new ticket in the [Governance][governance] repository
+  - The TA will create a new branch on the [Governance][governance] repository's ticket
+- The TA will modify the `config.yaml` file in this repository with the following information
+  - The new maintainer group name (`REPO-maintainers`)
+  - The new maintainers for the group
+  - The new committer group name (`REPO-committers`)
+  - the new set of committers for that group
```
Comment (andrewb1269hg) [inline hashgraph-transfer.md:81]: Capitalize `The` (the rest of the list has caps, this one does not)

**Review by andrewb1269hg — CHANGES_REQUESTED** [review]

File: hashgraph-transfer.md (hunk 2 above)
Comment (rbarkerSL) [reply hashgraph-transfer.md:57]:
```suggestion
    - List of Maven Central packages
 ```

File: hashgraph-transfer.md (hunk 3 above)
Comment (rbarkerSL) [reply hashgraph-transfer.md:63]:
```suggestion
    - Repository Point of Contact for day-of questions
```

File: hashgraph-transfer.md (hunk 5 above)
Comment (rbarkerSL) [reply hashgraph-transfer.md:67]:
```suggestion
the repository is ready for transfer.
```

File: hashgraph-transfer.md (hunk 7 above)
Comment (rbarkerSL) [reply hashgraph-transfer.md:81]:
```suggestion
  - The new set of committers for that group
```

File: hashgraph-transfer.md (hunk 6 above)
Comment (rbarkerSL) [reply hashgraph-transfer.md:48]:
```suggestion
## Day-Before Checks
```

**Review by andrewb1269hg — APPROVED** [review]

**Pull request merged**